### DISCUSSION TITLE
Can I use socks5 proxies?

### DISCUSSION THREAD

**Discussion by matheusbranhann** [KevinBytesTheDust/CVAmp#238] [Q&A]
i can use a socks 5?

**Comment by jlplenio** [comment]
I doubt it will work, as I hardcode the proxy list with a http prefix. You could edit this line and try, it should work.
https://github.com/jlplenio/crude-twitch-viewer-bot/blob/775a1f3de3305d610ae9ddf0f4e1e0ed130675d0/ctvbot/proxy.py#L40
